saleor/graphql/shipping/mutations/shippings.py

```diff
@@ -158,6 +158,79 @@ def success_response(cls, instance):
         return response
 
 
+class ShippingZipCodeRulesCreateInputRange(graphene.InputObjectType):
+    start = graphene.String(required=True, description="Start range of the zip code.")
+    end = graphene.String(required=False, description="End range of the zip code.")
+
+
+class ShippingZipCodeRulesCreateInput(graphene.InputObjectType):
+    zip_code_rules = graphene.List(
+        ShippingZipCodeRulesCreateInputRange, description="Start range of the zip code."
+    )
+
+
+class ShippingZipCodeRulesCreate(BaseMutation):
+    zip_code_rules = graphene.List(
+        ShippingMethodZipCodeRule, description="A shipping method zip code range.",
+    )
+    shipping_method = graphene.Field(
+        ShippingMethod, description="Related shipping method."
+    )
+
+    class Arguments:
+        shipping_method = graphene.ID(
+            required=True, description="ID of a shipping method to assign."
+        )
+        input = ShippingZipCodeRulesCreateInput(
+            description="Fields required to create a shipping zip codes.", required=True
+        )
+
+    class Meta:
+        description = "Create a new zip code exclusion range for shipping method."
+        permissions = (ShippingPermissions.MANAGE_SHIPPING,)
+        error_type_class = ShippingError
+        error_type_field = "shipping_errors"
+
+    @classmethod
+    def perform_mutation(cls, root, info, **data):
+        shipping_method = cls.get_node_or_error(
+            info, data["shipping_method"], only_type=ShippingMethod
+        )
+        instances = []
+        for zip_range in data["input"]["zip_code_rules"]:
```

Comment:
`zip_code_rules` are not required, so it will be raising an error in such a situation.
It seems like `zip_code_rules` should be required and set as `NonNull`.

Reply:
It should be required, changed it in input type, thanks.